### PR TITLE
Generate clear error for unconnected memory devices

### PR DESCRIPTION
**Pull request by slacrherbst**
This PR will ensure an error like below is generated if the user fails to properly connect the memory bus:

pyrogue._Block.MemoryError: 'Memory Error for dummyTree.AxiVersion.FpgaVersion at address 0x000000 Invalid min/max memory interface size. Device or Variable may be unconnected!'

### CODE REVIEW

File: src/rogue/interfaces/memory/Master.cpp [hunk 1]
```diff
@@ -68,7 +68,7 @@ void rim::Master::setup_python() {
 //! Create object
 rim::Master::Master() {
    error_   = "";
-   slave_   = rim::Slave::create(4,4); // Empty placeholder
+   slave_   = rim::Slave::create(4,0); // Empty placeholder
```
Comment (ruck314) [inline src/rogue/interfaces/memory/Master.cpp]: Why this change?

Comment (slacrherbst) [reply src/rogue/interfaces/memory/Master.cpp]: It sets the default Slave placeholder module for a master to have an invalid max size which triggers the error in the Block. Otherwise their is no way to tell if the memory device is not connected.
